sipa/blueprints/generic.py

```diff
@@ -162,7 +165,18 @@ def logout():
                 extra={'tags': {'user': current_user.uid}})
     logout_user()
     flash(gettext("Abmeldung erfolgreich!"), 'success')
-    return redirect(url_for('.index'))
+
+    resp = redirect(url_for('.index'))
+
+    cookie_name = current_app.config.get('REMEMBER_COOKIE_NAME', COOKIE_NAME)
+    if cookie_name in request.cookies:
+        resp.set_cookie(cookie_name, "", 0)
+
+    current_app.login_manager._clear_cookie(resp)
+    resp.set_cookie("session", "")
+    session.clear()
+
+    return resp
```

Comment:
I will have a look at your suggested solution. If there is a better way to implement the intended behavior I will try do it this way. But this may take some time. 

PS: I appreciate your constructive criticism